### PR TITLE
[bitnami/elasticsearch] Added master ingress

### PR DESCRIPTION
**Pull request by abhi1693**
**Description of the change**

Added master ingress configuration

**Benefits**

I needed to expose the master service so that my 3rd-party services could connect

**Possible drawbacks**

NA

**Applicable issues**

NA

**Additional information**

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/bitnami-docker-elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 17.6.1
+version: 17.6.2
```
Comment (javsalgar) [inline bitnami/elasticsearch/Chart.yaml:28]: Could you bump the minor version? It is adding a non-breaking feature

File: bitnami/elasticsearch/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+spec:
+  rules:
+    {{- if .Values.master.ingress.hostname }}
+    - http:
+        paths:
+          - path: {{ .Values.master.ingress.path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
+            pathType: {{ .Values.master.ingress.pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
```
Comment (javsalgar) [inline bitnami/elasticsearch/templates/ingress.yaml:29]: shouldn't serviceName be `include "elasticsearch.master.fullname" .`

File: bitnami/elasticsearch/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    {{- end }}
+    {{- range .Values.master.ingress.extraHosts }}
+    - host: {{ .name }}
+      http:
+        paths:
+          - path: {{ default "/" .path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" $) }}
+            pathType: {{ default "ImplementationSpecific" .pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" $) "servicePort" "http" "context" $) | nindent 14 }}
```
Comment (javsalgar) [inline bitnami/elasticsearch/templates/ingress.yaml:42]: Same here

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thank you so much for the PR! Changes requested

File: bitnami/elasticsearch/templates/ingress.yaml (hunk 2 above)
Comment (abhi1693) [reply bitnami/elasticsearch/templates/ingress.yaml:29]: This was an oversight on my part.

**Review by javsalgar — APPROVED** [review]
LGTM, thanks!

**Pull request merged**